crates/topos-tce-broadcast/src/double_echo/broadcast_state.rs

```diff
@@ -66,7 +66,7 @@ impl BroadcastState {
         });
 
         if need_gossip {
-            warn!("📣 Gossiping the Certificate {}", &state.certificate.id);
+            debug!("📣 Gossiping the Certificate {}", &state.certificate.id);
```

Comment:
It would be good to log the subnet involved here, and perhaps the validator id too?

Reply:
Yep, added subnet-id here https://github.com/topos-protocol/topos/pull/465/commits/7e4f32eb41aed838815ebfb6704f0cc78d816c28

As for the `validator-id`, I might prefer instead to have a robust and complete solution to expose the mapping `instance <> validator-id` for all the logs instead of sparkling some `validator-id` on some logs here and there no?